pkg/plugins/models.go

```diff
@@ -95,10 +95,19 @@ type PluginInclude struct {
 	DefaultNav bool            `json:"defaultNav"`
 	Slug       string          `json:"slug"`
 	Icon       string          `json:"icon"`
+	Uid        string          `json:"uid"`
```

Comment:
We should add this new field to the JSON schema https://github.com/grafana/grafana/blob/main/docs/sources/developers/plugins/plugin.schema.json#L58

Reply:
Nit:
```suggestion
	UID        string          `json:"uid"`
```